### ISSUE TITLE
Generator fails when attempting to make call to GitHub API's.

### ISSUE THREAD

**Issue by eborysko** [vajahath/generator-ts-np#54]
Testing out your generator here, but it fails when the calling GitHub API's... the user I operate under does not get GitHub account and I am underuse why the generator needs to call out to Github; especially if I'm attempting to make a private package on non-public repository.

```
What is your npm scope?  (without @)
  (If you want to publish at global scope, leave this field
  empty(or empty string ''). It is always nice to
  publish packages under a scope. Learn more about
  scopes here: https://docs.npmjs.com/misc/scope)
{ RequestError: connect ETIMEDOUT 192.30.253.116:443
    at ClientRequest.req.once.err (/Users/netsuite/.nvm/versions/node/v10.17.0/lib/node_modules/generator-ts-np/node_modules/gh-got/node_modules/got/index.js:73:21)
    at Object.onceWrapper (events.js:286:20)
    at ClientRequest.emit (events.js:198:13)
    at TLSSocket.socketErrorListener (_http_client.js:392:9)
    at TLSSocket.emit (events.js:198:13)
    at emitErrorNT (internal/streams/destroy.js:91:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  code: 'ETIMEDOUT',
  message: 'connect ETIMEDOUT 192.30.253.116:443',
  host: 'api.github.com',
  hostname: 'api.github.com',
  method: 'GET',
  path: '/search/users?q=XXXXXXXXXXXXXXXXXXXXXXXX%20in%3Aemail' }
```

**Comment by vajahath** [comment]
@eborysko thank you for reporting this.
The generator was attempting to guess your GitHub username by calling a GitHub API. This requires a fix. Let me try to fix this.

For transparency, this is what causing your issue: https://github.com/vajahath/generator-ts-np/blob/d63179a5e52258321e249d5732b82d54a46e6d86/base-structure/_meta/name-mapping.ts#L71

**Comment by vajahath** [comment]
fixed in v3.1.0. See the PR for knowing what changed, just in case you are interested. Thank you @eborysko

**Comment by vajahath** [comment]
@eborysko this issue is addressed and released in v3.2.0. Thanks.
